packages/blade/src/components/Button/Button/Button.tsx

```diff
@@ -105,6 +110,7 @@ const _Button: React.ForwardRefRenderFunction<BladeElementRef, ButtonProps> = (
       size={size}
       type={type}
       variant={variant}
+      intent={isAllowedIntent ? intent : undefined}
```

Comment:
<img width="1057" alt="image" src="https://github.com/razorpay/blade/assets/30949385/4ff66ba1-0aaf-491a-815f-ca706c7c47fa">

Can you check how we can make this work? Basically setting `variant` and `intent` both. So we get outlined button when variant is secondary. Might require some changes in BaseButton https://razorpay.slack.com/archives/G01B3LQ9H0W/p1690350610167359?thread_ts=1690347620.242949&cid=G01B3LQ9H0W

Reply:
@saurabhdaware From the discussion I recall that it is not something we want to allow.